spec/acceptance/manage_config_values_spec.rb

```diff
@@ -0,0 +1,170 @@
+require 'spec_helper_acceptance'
+
+confine(:to, :platform => 'windows')
+
+describe 'Chocolatey Config' do
+
+  context 'MODULES-3035 - Add New Config Item' do
+
+    before(:all) do
```

Comment:
Have you tried a "before(:each)" to avoid duplication of the before and after calls?

Reply:
I did. The problem is that each of the `it_behaves_like` blocks typically has two examples in it. One to apply the manifest, and the second to verify that the test property was applied correctly. If I do these as before and after each blocks, then my config file gets reset between each of those examples, and the second test will fail each time.